torchaudio/functional.py

```diff
@@ -1636,3 +1637,86 @@ def detect_pitch_frequency(
     freq = freq.view(shape[:-1] + list(freq.shape[-1:]))
 
     return freq
+
+
+def sliding_window_cmn_internal(
```

Comment:
Can you remove `_internal` from function name, as this will be user-facing function.

Reply:
ok